lib/tests/main.cc

```diff
@@ -4,31 +4,40 @@
 
 #include <catch.hpp>
 
+#include <boost/nowide/iostream.hpp>
```

Comment:
It's not needed; it got in by mistake for some i18n tests in another branch.

BTW, when you asked about this in the channel I thought you meant the configuration.cc one; please let's keep code conversation here.